src/common/regions/regions_data.ts

```diff
@@ -34,6 +34,9 @@ export function getStateCode(region: Region): string | null {
   if (region.regionType === RegionType.STATE) {
     return (region as State).stateCode;
   }
+  if (region.regionType === RegionType.MSA) {
```

Comment:
i just browsed around the uses of `getStateCode` and didnt catch anything that would break. did you see something?

Reply:
Hmm yeah I think you're right that I think it doesn't break anything. 

This is a bit of a semantic difference, but i'm thinking about how this changes the meaning of the function - even though the multiple state codes are pretty similar to state code they can be used for slightly different things: 
 * I would expect this function to return a single state code given a region from the function name, and that behavior is more undefined with Metros.  
 * From just reading the function, I definitely wouldn't expect it to return a hyphen separated string of multiple state codes - we pretty much made that up and if we keep this logic here, probably should rethink how we can communicate that
 * Since the state code is a pretty specific piece of data and is often (though much less often now) used to lookup states, I think we make other uses of "stateCode" a bit more confusing - maybe renaming this function would be better.

Anyways, I'm probably making this bigger deal than it is, just a bit worried that future us won't know the context for what a metro area "state code" is as we kind of made it up.

Reply:
would it help if i added documentation above the util?

Reply:
stateCode has a different meaning between all 3 region types and i think this function captures all of them relatively intuitively

Reply:
I think the `stateCode` has the same meaning between state and county, no?

If you want to add this logic here, I'd rather change the name of the function - it's a very small distinction, but the metro area state codes really isn't a state code and i'd prefer to not overload that term. This function was made outside of the `Region` type because Metros don't have a state code, otherwise this would've existed as `region.stateCode` to make life easier.

couple thoughts:
 * Can just do this logic in the `CompareTableRow` and punt on this issue for now.  
 * add a property `stateCodes` that returns a list of state codes on `Region` + maybe a function `formattedStateCodes` or something similar that returns the state code for County/State and `-` joined state codes for metro
 * Keep it as is in this PR - but probably add a trello ticket to take a look at this later.  Also, If we do this, need to update [this assert](https://github.com/covid-projections/covid-projections/blob/127207876074c2c9710eb666396bd4c1edfba97d/src/components/CountyMap/CountyMap.js#L18). Also, [this](https://github.com/covid-projections/covid-projections/blob/b8485690226f657a29b700acb5c0911e583fe09b/src/components/Newsletter/Newsletter.js#L77) behavior is a bit undefined (i'm not sure how the form uses the submitted state code) to check for state or county before giving it a state code.